packages/x-components/tests/unit/main-scroll.css

```diff
@@ -0,0 +1 @@
+
```

Comment:
nice blank line! I was just saying this is an empty file, but not! it has that nice blank line 😍

Reply:
If you remove this file the whole x-components break.